x-pack/plugins/fleet/public/services/package_verification.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { PackageInfo, PackageListItem } from '../types';
+
+import { ExperimentalFeaturesService } from '.';
+
+export function isPackageUnverified(
```

Comment:
it would be nice to add some unit tests on this function

Reply:
done 👍